x-pack/plugins/threat_intelligence/public/modules/indicators/lib/display_value.ts

```diff
@@ -56,6 +56,25 @@ const findMappingRule = (indicatorType: string | null): IndicatorValueExtractor
  */
 const rules: Record<string, IndicatorValueExtractor> = {};
 
+/**
+ * Mapping between the indicator type and the {@link RawIndicatorFieldId}.
+ */
+const indicatorTypeToField: { [id: string]: RawIndicatorFieldId } = {
+  file: RawIndicatorFieldId.FileSha256,
+  'ipv4-addr': RawIndicatorFieldId.UrlFull,
```

Comment:
I believe `ipv4-addr` and `ipv6-addr` should have replaced `ip` mapping, not the `url` one

Reply:
🤦